src/MagicLink.ts

```diff
@@ -50,21 +50,18 @@ class MagicLink extends CotterVerify {
     }
     ifrm.style.overflow = "scroll";
 
+    if(!this.config.RedirectURL) this.config.RedirectURL = window.location.href
+    this.config.SkipRedirectURL = true
```

Comment:
What happened if someone passed in a RedirectURL? (as in someone who is still using the old version like EVOS)? (they shouldn't be using the magic link so should be ok, but there might be other ppl?)

Reply:
Add deprecation on `RedirectURL`